src/pkg/inspection/scanner.go

```diff
@@ -160,33 +161,35 @@ func (ds *DefaultScanner) processPod(ctx context.Context, pod corev1.Pod, indexe
 	return nil
 }
 
-func (ds *DefaultScanner) getOwner(ctx context.Context, pod corev1.Pod) (*corev1.ObjectReference, error) {
+func (ds *DefaultScanner) getOwner(ctx context.Context, pod corev1.Pod) (*corev1.ObjectReference, int32, error) {
```

Comment:
This function now does 2 things: get owner and the replica count, so can we change the name of the function to make it clearer? Or, is it better to seperate into 2 functions?

Reply:
Anyway, this need to be resolved when we merge the code to main branch in the future.

WRT merging this change to feature-governer, now it is ok to write like that 😄